habitat/tasks/nav/semantic_constants.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+GIBSON_CATEGORY_TO_TASK_CATEGORY_ID = {
+    'chair': 0,
+    'dining table': 1,
+    'book': 2,
+    'vase': 3,
+    'bottle': 4,
+    'couch': 5,
+    'bed': 6,
+    'refrigerator': 7,
+    'potted plant': 8,
+    'sink': 9,
+    'toilet': 10,
+    'clock': 11,
+    'towel': 12,
+    'tv': 13,
+    'oven': 14,
+    'cup': 15,
+    'umbrella': 16,
+    'bowl': 17,
+    'gym_equipment': 18,
+    'bench': 19,
+    'clothes': 20
+}
+
+
+MP3D_CATEGORY_TO_TASK_CATEGORY_ID = {
```

Comment:
This is different than https://github.com/niessner/Matterport/blob/master/metadata/mpcat40.tsv which will likely lead to confusion.